delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/StoragesConfig.scala

```diff
@@ -206,7 +207,7 @@ object StoragesConfig {
       defaultWritePermission: Permission,
       showLocation: Boolean,
       defaultMaxFileSize: Long,
-      digestComputation: RetryStrategyConfig
+      digestComputationRetryDelay: FiniteDuration
```

Comment:
Here, we always want to retry so it is better to just leave the opportunity to define a delay between them.